packages/api/src/record.ts

```diff
@@ -304,14 +444,55 @@ export class Record implements RecordModel {
    *
    * @beta
    */
```

Comment:
Please update the explanation for what `send()` does because it isn't the same as before.